force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -80,16 +99,23 @@ public with sharing class ClassesObjectsMethods {
      * @return A string consisting of only unique words from the input sentence.
      */
     public static String removeDuplicateWords(String sentence) {
+       
         // Split the sentence into words
+        List<String> words = sentence.split(' ');
 
         // Create a set to store the unique words
+        Set<String> wordSet = new Set<String>();
 
         // Add each word to the set
+        for (String word : words) {
+            wordSet.add(word);
+        }
 
         // Join the unique words back into a sentence
+        String newSentence = String.join(wordSet, ' ');
```

Comment:
This is a great way to solve but also try using Set.addAll

Reply:
Yup! Can't believe I missed that, very slick. RTFM right?